plaso/parsers/presets.py

```diff
@@ -3,11 +3,12 @@
 
 from __future__ import unicode_literals
 
+# TODO: add gdrive_sync to linux/macos after further testing.
```

Comment:
What is the status of this?

Reply:
Pending, it's not available for Linux externally and I don't have a Mac. I will probably be able to get a sample log generated next week for Mac, I'd think -- shall I create an issue and assign it to myself?

Reply:
yes please do

Reply:
https://github.com/log2timeline/plaso/issues/1712